lib/job-iteration/iteration.rb

```diff
@@ -112,15 +112,15 @@ def serialize # @private
       super.merge(
         "cursor_position" => cursor_position,
         "times_interrupted" => times_interrupted,
-        "total_time" => total_time,
+        "total_time" => total_time.to_f,
```

Comment:
This one should not be necessary if we're always casting it on deserialization, right?

Reply:
Yeah, not necessary. Simply added for completeness.